

============================== 2023-12-07 14:41:59.743822 | 2fa6af2c-d360-4c9a-b292-b0a6e66510cb ==============================
[0m14:41:59.743822 [info ] [MainThread]: Running with dbt=1.4.9
[0m14:41:59.745945 [debug] [MainThread]: running dbt with arguments {'write_json': True, 'use_colors': True, 'printer_width': 80, 'version_check': True, 'partial_parse': True, 'static_parser': True, 'profiles_dir': '/Users/mac/.dbt', 'send_anonymous_usage_stats': True, 'quiet': False, 'no_print': False, 'cache_selected_only': False, 'project_name': 'dagster_dbt', 'skip_profile_setup': False, 'which': 'init', 'indirect_selection': 'eager'}
[0m14:41:59.746248 [debug] [MainThread]: Tracking: tracking
[0m14:41:59.771950 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108991a10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104b75a10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1089b6550>]}
[0m14:41:59.773448 [info ] [MainThread]: Creating dbt configuration folder at /Users/mac/.dbt
[0m14:41:59.775403 [debug] [MainThread]: Starter project path: /Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/include/starter_project
[0m14:42:03.980866 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10901fb10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10901f210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10901f310>]}
[0m14:42:03.981638 [debug] [MainThread]: Flushing usage events
[0m14:42:05.556449 [error] [MainThread]: Encountered an error:
dlopen(/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/pyodbc.cpython-311-darwin.so, 0x0002): Library not loaded: /usr/local/opt/unixodbc/lib/libodbc.2.dylib
  Referenced from: <D23A4D34-B79E-35E4-88C9-80F238373822> /Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/pyodbc.cpython-311-darwin.so
  Reason: tried: '/usr/local/opt/unixodbc/lib/libodbc.2.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/opt/unixodbc/lib/libodbc.2.dylib' (no such file), '/usr/local/opt/unixodbc/lib/libodbc.2.dylib' (no such file), '/usr/lib/libodbc.2.dylib' (no such file, not in dyld cache)
[0m14:42:05.568472 [error] [MainThread]: Traceback (most recent call last):
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/main.py", line 136, in main
    results, succeeded = handle_and_check(args)
                         ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/main.py", line 206, in handle_and_check
    task, res = run_from_args(parsed)
                ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/main.py", line 253, in run_from_args
    results = task.run()
              ^^^^^^^^^^
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/task/init.py", line 305, in run
    self.create_profile_from_target(adapter, profile_name=project_name)
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/task/init.py", line 173, in create_profile_from_target
    load_plugin(adapter)
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/adapters/factory.py", line 202, in load_plugin
    return FACTORY.load_plugin(name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/adapters/factory.py", line 57, in load_plugin
    mod: Any = import_module("." + name, "dbt.adapters")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/adapters/sqlserver/__init__.py", line 3, in <module>
    from dbt.adapters.sqlserver.sql_server_adapter import SQLServerAdapter
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/adapters/sqlserver/sql_server_adapter.py", line 13, in <module>
    from dbt.adapters.sqlserver.sql_server_connection_manager import SQLServerConnectionManager
  File "/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/adapters/sqlserver/sql_server_connection_manager.py", line 10, in <module>
    import pyodbc
ImportError: dlopen(/Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/pyodbc.cpython-311-darwin.so, 0x0002): Library not loaded: /usr/local/opt/unixodbc/lib/libodbc.2.dylib
  Referenced from: <D23A4D34-B79E-35E4-88C9-80F238373822> /Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/pyodbc.cpython-311-darwin.so
  Reason: tried: '/usr/local/opt/unixodbc/lib/libodbc.2.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/opt/unixodbc/lib/libodbc.2.dylib' (no such file), '/usr/local/opt/unixodbc/lib/libodbc.2.dylib' (no such file), '/usr/lib/libodbc.2.dylib' (no such file, not in dyld cache)



============================== 2023-12-07 14:48:46.977319 | 6fbd7e77-02c8-46ec-afad-872c76099069 ==============================
[0m14:48:46.977319 [info ] [MainThread]: Running with dbt=1.4.9
[0m14:48:46.979859 [debug] [MainThread]: running dbt with arguments {'write_json': True, 'use_colors': True, 'printer_width': 80, 'version_check': True, 'partial_parse': True, 'static_parser': True, 'profiles_dir': '/Users/mac/.dbt', 'send_anonymous_usage_stats': True, 'quiet': False, 'no_print': False, 'cache_selected_only': False, 'project_name': 'dagster_dbt', 'skip_profile_setup': False, 'which': 'init', 'indirect_selection': 'eager'}
[0m14:48:46.980138 [debug] [MainThread]: Tracking: tracking
[0m14:48:47.017845 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107bda450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108237810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108237b50>]}
[0m14:48:47.020611 [info ] [MainThread]: A project called dagster_dbt already exists here.
[0m14:48:47.021170 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107c7b450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107f377d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107bb2590>]}
[0m14:48:47.021542 [debug] [MainThread]: Flushing usage events


============================== 2023-12-08 10:23:40.270272 | f63aa9c0-d588-4227-8dc8-d0f414c6d4c6 ==============================
[0m10:23:40.270272 [info ] [MainThread]: Running with dbt=1.4.9
[0m10:23:40.272726 [debug] [MainThread]: running dbt with arguments {'write_json': True, 'use_colors': True, 'printer_width': 80, 'version_check': True, 'partial_parse': True, 'static_parser': True, 'profiles_dir': '/Users/mac/.dbt', 'send_anonymous_usage_stats': True, 'quiet': False, 'no_print': False, 'cache_selected_only': False, 'project_name': 'dag_eng', 'skip_profile_setup': False, 'which': 'init', 'indirect_selection': 'eager'}
[0m10:23:40.273404 [debug] [MainThread]: Tracking: tracking
[0m10:23:40.303903 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108c6e710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108febfd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108febf90>]}
[0m10:23:40.308447 [debug] [MainThread]: Starter project path: /Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/lib/python3.11/site-packages/dbt/include/starter_project
[0m10:23:43.227408 [info ] [MainThread]: No sample profile found for sqlserver.
[0m10:23:43.229175 [info ] [MainThread]: 
Your new dbt project "dag_eng" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:23:43.230700 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1088fb4d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10893f950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108c06d50>]}
[0m10:23:43.232422 [debug] [MainThread]: Flushing usage events
[0m10:23:43.255432 [debug] [MainThread]: Error sending anonymous usage statistics. Disabling tracking for this execution. If you wish to permanently disable tracking, see: https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats.
[0m12:49:00.225846 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105428390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1003b5f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10542ac10>]}


============================== 12:49:00.236428 | 6be745c0-0854-4008-9872-8e4498a23cc6 ==============================
[0m12:49:00.236428 [info ] [MainThread]: Running with dbt=1.7.3
[0m12:49:00.237698 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/Users/mac/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m12:49:00.238677 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/mac/Desktop/Analytics_Enginerring/dagster-dbt/.env/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:49:00.240830 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.047216423, "process_user_time": 2.840563, "process_kernel_time": 0.375501, "process_mem_max_rss": "107286528", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:49:00.241805 [debug] [MainThread]: Command `dbt run` failed at 12:49:00.241568 after 0.05 seconds
[0m12:49:00.242459 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1050d7e90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10034ead0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1050e8890>]}
[0m12:49:00.243125 [debug] [MainThread]: Flushing usage events
